doc/source/serve/multi-app.md

```diff
@@ -0,0 +1,208 @@
+# Deploying Multiple Serve Applications
+
+In Ray 2.4+, deploying multiple independent Serve applications is supported. This user guide walks through how to generate a multi-application config file and deploy it using the Serve CLI, and monitor your applications using the CLI and the Ray Serve dashboard.
+
+## When to Use Multiple Applications
+It can be useful to deploy multiple independent applications on one Ray cluster. However, it is also important to keep in mind the downsides of doing so.
```

Comment:
It's not very clear what the downsides are. eg. what would the competition for resources lead to?

What are the main reasons we would not want people to use multi-app?

Reply:
Added a part talking about starvation of resources. Got approval from @akshay-anyscale offline.

@edoakes @sihanwang41 Could you also take a look at this section? I wrote this based off of my understanding, I think it'd be helpful to get your guys' input as well. This is going to be at the top of the page.

Reply:
I would prefer to directly remove the downside part. 
The downside mentioned here is not very clear for user to decide whether to use single app or multi-app. 
- Sharing resource is very vague topic. (e.g. there might be also latency wise perspective to consider, it is more case by case.)
- Isolation can be achieved by setting up some resource tags within one cluster with different nodes.
These trade offs might be better fit for user case blog stuff :).
For day1 user, I am concerned it will cause some confusion.

Reply:
Resolved offline.